### DISCUSSION TITLE
jsonTable

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1079] [Q&A]
Style bezieht sich bislang auf Breite von der Spalte. Man kann aber über Columns-Einstellungen keine Rahmen setzen.
![image](https://user-images.githubusercontent.com/84818934/135842526-133e995e-700f-41cb-ab2f-b08a6b945c2b.png)
Das kann man nur mit dem content. 
![image](https://user-images.githubusercontent.com/84818934/135842630-515dc9f1-065d-40f8-b309-419b04952607.png)

[{....,"separator":"cell"},{....}] funktioniert nicht.
Wie komme ich daran? 
Oder wie muss man es einstellen im Array?

**Comment by Zefau** [comment]
Hast du im `style` mal `border` probiert?

**Reply by mcuiobroker** [reply]
Ich hab es mit "bordered":true und auch "border":true versucht:
`[  {    "field": "Nr",    "label": "Nr",    "style": "width:50px;background-color: #ff0000"  },  {    "field": "ZEIT",    "label": "Zeit",    "style": "width:50px; bordered:true" },  {    "field": "KANAL",    "label": "Kanal",    "style": "width:50px"  },  {    "field": "TITEL",    "label": "Titel"  }] `

**Comment by mcuiobroker** [comment]
wrap-cells ist auch ein Thema, da man auch in Zellen längere Texte hat.
![image](https://user-images.githubusercontent.com/84818934/136182683-846544ed-c8a7-447e-a500-318328b8a070.png)

Soll ich mal dazu ein Feature Request aufmachen?
